src/type/enum.tsx

```diff
@@ -0,0 +1,12 @@
+export enum EnumPeoples {
+  'Luke Skywalker' = '/assets/People/Luke_Skywalker.jpg',
+  'C-3PO' = '/assets/People/C-3PO.jpg',
+  'R2-D2' = '/assets/People/R2-D2.png',
+  'Darth Vader' = '/assets/People/Darth_Vader.png',
+  'Leia Organa' = '/assets/People/Leia_Organa.jpg',
+  'Owen Lars' = '/assets/People/Owen_Lars.jpg',
+  'Beru Whitesun lars' = '/assets/People/Beru_Whitesun_lars.jpg',
+  'R5-D4' = '/assets/People/R5-D4.webp',
+  'Biggs Darklighter' = '/assets/People/Biggs_Darklighter.jpg',
+  'Obi-Wan Kenobi' = '/assets/People/Obi-Wan_Kenobi.jpg',
+}
```

Comment:
use javascript compatible names

```suggestion
export enum EnumPeoples {
  LukeSkywalker = '/assets/People/Luke_Skywalker.jpg',
  C3PO = '/assets/People/C-3PO.jpg',
  R2D2 = '/assets/People/R2-D2.png',
  DarthVader = '/assets/People/Darth_Vader.png',
  LeiaOrgana = '/assets/People/Leia_Organa.jpg',
  OwenLars = '/assets/People/Owen_Lars.jpg',
  BeruWhitesunlars = '/assets/People/Beru_Whitesun_lars.jpg',
  R5D4 = '/assets/People/R5-D4.webp',
  BiggsDarklighter = '/assets/People/Biggs_Darklighter.jpg',
  ObiWanKenobi = '/assets/People/Obi-Wan_Kenobi.jpg',
}
```

Reply:
Bad naming and u don't use this enum anywhere